marconi-core/src/Marconi/Core/Experiment/Indexer/SQLiteIndexer.hs

```diff
@@ -302,6 +302,9 @@ instance (MonadIO m) => IsSync m event SQLiteIndexer where
   lastSyncPoint indexer =
     pure $ indexer ^. dbLastSync
 
+-- TODO This often results in the runtime error:
+-- SQLite3 returned ErrorBusy while attempting to perform close: unable to close due to unfinalized statements or unfinished backups
+-- Should use 'withConnection' instead.
```

Comment:
It may not be related to the behaviour of the SQLite indexer but to the test framework, so maybe we can drop the comment?